src/common/translation.cpp

```diff
@@ -1451,12 +1452,70 @@ wxString wxTranslations::DoGetBestAvailableTranslation(const wxString& domain, c
     return lang;
 }
 
+namespace {
+
+/* As of October 2025, MinGW has a bug regarding thread_local
+ * variables: their memory is de-allocated, and then their destructor
+ * is called.
+ *
+ * The UntranslatedStringHolder class works around this issue, having
+ * its destructor access no data members.
+ */
+#if defined __MINGW32__ || defined __MINGW64__
```

Comment:
I think `__MINGW32__` is defined when using 64-bit MinGW too, so the second check is redundant.